contrib/resources/drives/y/3dfx/license.txt

```diff
@@ -0,0 +1,96 @@
+3DFX INTERACTIVE, INC. LICENSE AGREEMENT 
+
+LICENSE AND CONFIDENTIALITY:  3dfx Interactive, Inc. ("3dfx") grants you the
+right to install the enclosed software and related documentation (collectively,
+the "Materials") onto a single computer for your personal use.  You may not use,
+copy, modify, sell, transfer or disclose any part of the Materials except as
+provided in this Agreement. You may only use the Materials in connection with
+3dfx products.
+ 
+RESTRICTIONS:  You may not:
+1.	Sublicense the Materials;
+2.	Reverse engineer, decompile, or disassemble the enclosed software;
+3.	Use the Materials for any platform or products other
```

Comment:
> You may only use the Materials in connection with 3dfx products
> You may not (…) Use the Materials for any platform or products other than 3dfx products

Please do not include anything with such a restrictive license, or we will get kicked out of every single Linux distro, not to mention there is a non-zero chance of legal problems for the project.

If you really want to ship Glide with DOSBox, the best way would be to compile it from the sources https://github.com/sezero/glide - after studying the license https://github.com/sezero/glide/blob/glide-devel-sezero/LICENSE

Reply:
If you install a 3DFX compatible DOS game from the CD-ROM (ISO, BIN CUE or original disc), it usually has the Voodoo 1 compatible `glide.ovl` driver, so you may not need all this.

Reply:
> we will get kicked out of every single Linux distro

DOSBox-X encodes glide2x.ovl [in a header](https://github.com/joncampbell123/dosbox-x/blob/master/src/builtin/glide2x.h).

This hasn't stopped the [AUR](https://aur.archlinux.org/packages/dosbox-x), [Debian](https://sources.debian.org/src/dosbox-x/2023.10.06+dfsg-1/), [FreeBSD](https://www.freshports.org/emulators/dosbox-x), [Gentoo overlays](https://gpo.zugaina.org/Overlays/guru/games-emulation/dosbox-x), [Homebrew](https://formulae.brew.sh/formula/dosbox-x), [MacPorts](https://ports.macports.org/port/dosbox-x/summary/), [NixOS](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/emulators/dosbox-x/default.nix), [Mandrake](https://github.com/OpenMandrivaAssociation/dosbox-x/tree/master), Raspbian (via Debian), [Slack](https://slackbuilds.org/repository/15.0/system/dosbox-x/), and [Termux](https://github.com/termux/termux-packages/tree/51a389360/x11-packages/dosbox-x) repository maintainers from bundling DOSBox-X:

But more specifically, repo maintainers will kick-out content they they don't like to keep a package.

For example, Debian's maintainer has 11 code patches that strip various features from DOSBox-X including: playing MIDI through the bundled FluidSynth, using the handful of freeDOS binaries, and even using FLAC and Opus audio tracks :grimacing: 

---

Even if we're OK with the above approach, I don't want fearful repo maintainers crippling the user experience, so I will attempt the compiled approach.

Thanks @FeralChild64!

Reply:
Dropped, will re-add using the compiled approach.

Reply:
Distro maintainers need bigger balls 😎